### DISCUSSION TITLE
Winlogbeat Not Connecting to Logstash SO 2.3.50

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#4106] [Unsupported Versions] [answered]
Hello,

After deploying Winlogbeat to several hosts Logstash on Security Onion will not connect. I've allowed the subnet range through SO firewall by running sudo so-allow, choosing option B and inputting the correct subnet range. Any ideas why winlogbeat will not connect to Logstash? I've confirmed IP's are correct and checked all hosts to see if they are attempting to connect to the correct IP as well.

Running SO 2.3.50
Winlogbeat 7.11.2

**Comment by weslambert** [comment, marked as answer]
Is Logstash running on the Security Onion node?  Have you verified local connectivity to 5044, or are you able to use `nc` or Powershell's `Test-NetConnection` to ensure the port is listening?

**Reply by ColeVan** [reply]
@weslambert I was able to remedy the issues with a fresh install of SO 2.3.50. No issues. Thank you for your reply and feedback.
